### ISSUE TITLE
no-standalone-expect: false warning for `test` alias

### ISSUE THREAD

**Issue by EvgenyOrekhov** [playwright-community/eslint-plugin-playwright#251]
Example [from the docs](https://playwright.dev/docs/auth#basic-shared-account-in-all-tests):

```ts
import { test as setup, expect } from '@playwright/test';

setup('authenticate', async ({ page }) => {
  await expect(page.getByRole('button', { name: 'View profile and more' })).toBeVisible();
});
```

Expected: no warning

Actual: "Expect must be inside of a test block" warning

eslint-plugin-playwright version: 1.3.1
Node.js version: 18.19.0
